sdk/examples/xo_python/sawtooth_xo/xo_client.py

```diff
@@ -69,22 +78,24 @@ def list(self):
         except BaseException:
             return None
 
-    def show(self, name):
+    def show(self, name, auth_user=None, auth_password=None):
         address = self._get_address(name)
 
-        result = self._send_request("state/{}".format(address), name=name)
-
+        result = self._send_request("state/{}".format(address), name=name,
+                                    auth_user=auth_user,
+                                    auth_password=auth_password)
         try:
             return base64.b64decode(yaml.safe_load(result)["data"])
 
         except BaseException:
             return None
 
-    def _get_status(self, batch_id, wait):
+    def _get_status(self, batch_id, wait, auth_user=None, auth_password=None):
         try:
             result = self._send_request(
-                'batch_status?id={}&wait={}'.format(batch_id, wait)
-            )
+                'batch_status?id={}&wait={}'.format(batch_id, wait),
+                auth_user=None,
```

Comment:
This should be passing the `auth_user`, `auth_password` from the args to this call.